spec/controllers/user_targets_controller_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+RSpec.describe UserTargetsController, type: :controller do
+  render_views
+
+  describe 'GET #index' do
+    let!(:user) { FactoryGirl.create(:user) }
+    let!(:topic) { FactoryGirl.create(:topic) }
+    let!(:target) { FactoryGirl.create(:user_target, user: user, topic: topic) }
+
+    before(:each) do
+      user.confirm
+    end
+
+    context 'with user signed in' do
+      before(:each) do
+        sign_in user
+        get :index, format: :json
+      end
+
+      it 'responds HTTP 200' do
+        expect(response).to have_http_status(200)
+      end
+
+      it 'renders index' do
+          expect(response).to render_template('index')
+      end
+
+      it 'respond with correct format' do
```

Comment:
You aren't testing the format of the answer here, you're testing the answer data itself. Create two separate tests, one for the format using `expect(response.content_type).to eq 'application/json'`, and the other will be the one you've already written but with different description.